demo/src/entry-client.jsx

```diff
@@ -0,0 +1,4 @@
+// import { hydrate } from 'preact';
+// import { App } from './App';
+
+// hydrate(<App />, document);
```

Comment:
This is currently commented out, uncommenting this will make the `CommentNode` dissapear for the closing comment and hence kill hydration